src/encoded/types/acl.py

```diff
@@ -15,10 +13,10 @@
 #           This controls roles associated with particular item types based on properties they have
 #           ie: consortia, submission center presence indicates permissions (roles) should be given
 #           to a user
-CONSORTIUM_MEMBER_CREATE = 'role.consortium_member_create'
-SUBMISSION_CENTER_MEMBER_CREATE = 'role.submission_center_member_create'
-CONSORTIUM_MEMBER_RW = 'role.consortium_member_rw'
-SUBMISSION_CENTER_RW = 'role.submission_center_member_rw'
+CONSORTIUM_MEMBER_CREATE = "role.consortium_member_create"
+SUBMISSION_CENTER_MEMBER_CREATE = "role.submission_center_member_create"
+CONSORTIUM_MEMBER_RW = "role.consortium_member_rw"
+SUBMISSION_CENTER_RW = "role.submission_center_member_rw"
 
```

Comment:
What linter are you using? We should standardize this as a make target so formatting can be done automatically so we don't get large diffs across the repo like this that are pointless.